edit_tiles/app.py

```
@@ -3779,20 +3808,135 @@ def open_base_graph_dialog(self):
+                        "Origin longitude (-180 to 180):",
+                        initialvalue=f"{self.map_origin_lon:.8f}",
+                    )
+                    if longitude_text is None:
+                        return
+                    try:
+                        map_origin_lat = float(latitude_text.strip())
+                        map_origin_lon = float(longitude_text.strip())
+                    except ValueError:
+                        raise ValueError(
+                            "Map origin latitude and longitude must be numbers"
+                        )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)